webpack/components/extensions/RegistrationCommands/fields/IgnoreSubmanErrors.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'foremanReact/common/helpers';
+
+import { FormGroup, Checkbox } from '@patternfly/react-core';
+import LabelIcon from 'foremanReact/components/common/LabelIcon';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const IgnoreSubmanErrors = ({ value, onChange, isLoading }) => (
+  <FormGroup fieldId="reg_katello_ignore">
+    <Checkbox
+      label={
+        <span>
+          {__('Ignore errors')}{' '}
```

Comment:
Does the functionality ignore just subman errors or any error raised while registering? Just want to be sure that we are being clear. In general I don't think we should refer to the name of other tooling (subman here) if we can help it - that may change one day (and perhaps soon!)

Reply:
This is only for subscription-manager errors